node/src/runtime/mainnet.rs

```diff
@@ -24,7 +24,7 @@ impl ValidatorIdentity {
 		}
 	}
 
-	pub fn from_pub(ed: &str, sr: &str) -> ValidatorIdentity {
+	pub fn from_pub(ed: &str, sr: &str, _ecdsa: &str) -> ValidatorIdentity {
```

Comment:
it's definitely needed?

Reply:
ahh... I got it

Reply:
yes, because we need beefy keys  in future. 
and mainnet/testnet call the same interface "ValidatorIdentity::from_pub(" in the node/src/chain_spec.rs 

If you use testnet/mainnet flag macros need to generate more redundant code, it is no good